collector/collector.go

```diff
@@ -691,6 +668,17 @@ func (c *collector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error)
 		return nil, fmt.Errorf("No metrics found")
 	}
 
+	// add labels as tags to metrics
+
+	for i := range metrics {
+		rid := metrics[i].Namespace[2].Value
+		if rid != "root" {
+			// add labels only for docker cointainer, skip the host
+			metrics[i].Tags = c.containers[rid].Specification.Labels
```

Comment:
Removing assignment [Tags: mt.Tags](https://github.com/intelsdi-x/snap-plugin-collector-docker/pull/68/files#diff-3f2a717db5d6f646b50b72c60b0117f6L195) currently cause that metrics[i].Tags is nil, so metrics[i].Tags["some_key"] will cause panic. Realization of @PatrykMatyjasek's suggestion is only possible together with rollback changes introduced by commit [6fdbcb7](https://github.com/intelsdi-x/snap-plugin-collector-docker/pull/68/commits/6fdbcb70a3f33181e517c7c0c0878d9a3cf803a0) which is in the scope of this PR. To be clear, that is also my preference.